assets/templateFloxEnv/manifest.toml

```diff
@@ -9,20 +9,31 @@
 
 [install]
 _FLOX_INIT_INSTALL
-# Set an environment variable.
-# These variables may not reference once another
-# and are added to the environment without expansion.
+# Set environment variables in the 'vars' section.
+# These variables may not reference once another, and are added to
+# the environment without first expanding them.
 
 [vars]
 # message = "Howdy"
-# pass-in = "$some-env-var"
-
-# An activation hook will be run when entering the environment.
-# You can define one in the 'hook' table inline via the 'script' field,
-# or provide a file using the 'file' field.
-# If 'file' is provided, the path should be relative to this file.
-# If both 'file' and 'script' are provided,
-# only the hook defined in 'file' will be run .
+
+# Profile scripts are sourced by your shell immediately after entering the
```

Comment:
Would be good to note if `[vars]` are available here in `[profile]` scripts.  I see later it is defined in the docs, but since it's noted for `[hook]`, probably good to note it here as well?